container-compose.yml

```diff
@@ -46,6 +46,8 @@ services:
         source: orbit-db
         target: /var/orbit
         read_only: false
+    depends_on:
```

Comment:
maybe this should be the other direction? like we need orbit to come up before submatrix can as opposed to the opposite

Reply:
yeah that's fair